components/service/sync-logins/build.gradle

```diff
@@ -29,13 +29,18 @@ android {
 
 dependencies {
     // Parts of this dependency are typealiase'd or are otherwise part of this module's public API.
-    api Dependencies.mozilla_sync_logins
+    api(Dependencies.mozilla_sync_logins) {
+      // Use our own version of the Glean dependency,
+      // which might be newer than the version declared by A-S.
+      exclude group: 'org.mozilla.components', module: 'service-glean'
+    }
```

Comment:
I copied this from the existing rusthttp integration, in which a-s takes a dependency on a-c to get at concept-fetch.

Reply:
nit: 'newer' is probably better phrased as 'different'.

Reply:
Heh, I don't think it will ever be older, but I agree that's probably clearer.